### PR TITLE
Bug 1157496 - Should clear the previous double tapping timer when we touch a different target.

### PR DESCRIPTION
**Pull request by RudyLu**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cf4aeab9d3a03608a5e2f02bdc954d4ec6c9c908

File: apps/keyboard/js/keyboard/active_targets_manager.js [hunk 1]
```diff
@@ -243,6 +243,10 @@ ActiveTargetsManager.prototype._handlePressEnd = function(press, id) {
       this.ontargetdoubletapped(target);
     }
   } else {
+    if (this.doubleTapTimer) {
```
Comment (timdream) [inline apps/keyboard/js/keyboard/active_targets_manager.js]: Remove this `if` block.

Comment (timdream) [inline apps/keyboard/js/keyboard/active_targets_manager.js]: Please also do `this.doubleTapPreviousTarget = null;` at line 241

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=88181ebeceda5160a25a0ad0466f369844346344

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#5XQcu1vGQai3kNIcs5-z8w

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=af50c34a5da9066c2b698c83b372a13d95f6b93b

**Pull request merged**